week5/will/taobao/index.css

```diff
@@ -0,0 +1,296 @@
+body {
+  margin: 0px;
+}
+
+a {
+  text-decoration: none;
+  font-size: 12px;
+  color: #6c6c6c;
+  font-family: "Microsoft YaHei", tahoma, arial, "Hiragino Sans GB",
+    "\5b8b\4f53", sans-serif;
+}
+
+a:hover {
+  color: #f40;
+}
+
+.login-footer-container a:hover {
+  color: #f40;
+}
+
+nav img {
+  width: 150px;
+  margin: 25px 20px 15px 0px;
+}
+
+nav p {
+  margin-top: 28px;
+  font-size: 25px;
+  font-weight: 100;
+}
+
+nav {
+  display: flex;
+  align-items: center;
+  width: 990px;
+  margin: auto;
+}
+
+hr {
+  border: 1px solid #eee;
+}
+
+section {
+  width: 990px;
+  display: flex;
+  justify-content: flex-end;
+  margin: auto;
+}
+
+.login-container {
+  border: 1px solid #e8e8e8;
+  border-top: 2px solid #f40;
+  box-shadow: 0 2px 6px #acacac;
+  width: 350px;
+  height: 395px;
+  margin-top: 35px;
+  z-index: 0;
+  position: relative;
+}
+
+.login-container img {
+  width: 200px;
+  position: absolute;
+  right: 3px;
+  top: 15px;
+  z-index: 1;
+}
+
+.login-block {
+  z-index: 2;
```

Comment:
there's no position for this element, why are you using zindex?

Reply:
fixed